### ISSUE TITLE
Blobs are immune to freon

### ISSUE THREAD

**Issue by IcePacks** [tgstation/tgstation#30919]
[Directions]: # https://atlantaned.space/newSS13tools/round.php?round=76385
step 1) be a blob
step 2) spawn in engineering storage, destroying emitters and freon
step 3) freon everywhere
step 4) you are immune to freon
step 4b) humans are, barring some edge cases, not, and unable to siphon it
step 5) win

In the round attached, literally nobody was able to approach the blob, which won with no threats whatsoever the entire round.

**Comment by Cyberboss** [comment]
Was that so hard? :^)

**Comment by IcePacks** [comment]
yes

**Comment by Repukan** [comment]
Aren't ranged laser projectiles a thing?

**Comment by IcePacks** [comment]
they couldn't be for all you know

the primary issue here is that it was impossible to approach the blob at all considering the engine hallway and maint was full of the stuff

**Comment by Repukan** [comment]
I mean air scrubbing and deconstructing the floors are a thing.

**Comment by IcePacks** [comment]
air scrubbers do not do anything against plasma, good luck deconstructing every single floor tile while you're being harassed by blobbernauts and spores (or logging in the game to test any more solutions here)

can you cast doubt on an issue that you can actually influence or be influenced by instead of haunting githubs on games you're permabanned from

**Comment by Repukan** [comment]
Me being permabanned has nothing to do with probing your claim that nothing could be done to deal with the freon in your situation.

**Comment by IcePacks** [comment]
since you're unable to play and i was observing the round just take my word for it okay????

**Comment by Repukan** [comment]
That's stupid

**Comment by ShizCalev** [comment]
https://i.imgur.com/rwqDCbI.gifv

No issue with scrubbing freon with air scrubbers here.

**Comment by ShizCalev** [comment]
From what it sounds like to me, this really isn't a bug/issue since the players didn't know that they are able to siphon freon, and the blob just had a really good strategy.

**Comment by kevinz000** [comment]
alllriiiight
lisssteeeeen.
FREON IS NOT EVEN THAT POWERFUL IN A BLOB FIGHT.
Turn off gravity, most of its power is being horrible with its slipping and requiring a hardsuit in higher amounts due to supercooling the air. When gravity is off, slipping wont' happen because you can't slip without gravity (even with magboots/jetpack!). So, that makes it as deadly as a blob surrounded by space.

tl;dr Deal with it, I thought this was OP then I realized just turn the gravity breaker off to cripple freon strategy.

**Comment by kevinz000** [comment]
@optimumtact if this is still going to be considered an issue please say what part of it is an issue, like the blobs not taking damage, or the blobs not getting frozen and shattering when thrown (lol like that's ever going to happen outside of meteorshot), or what?

**Comment by Armhulen** [comment]
icepacks made this account just to post this hahahahaha

**Comment by bgobandit** [comment]
milestone: Goofball, Fix Your Shit @Iamgoofball

**Comment by CPTANT** [comment]
This shit again? We already had 2 of these:
#29421 
#29275

**Comment by ExcessiveUseOfCobblestone** [comment]
Kevinz talking trash on here but posted the exact same issue really activates the almonds.

**Comment by duncathan** [comment]
@as334 mark this as resolved in assmos pr lul

**Comment by evsey9** [comment]
close pr, assmos got merged

**Comment by Iamgoofball** [comment]
Somethings wrong, i hold my head

**Comment by PKPenguin321** [comment]
Freon gone
Server crashes dead
